### ISSUE TITLE
Linux consumption Metrics

### ISSUE THREAD

**Issue by balag0** [Azure/Azure-Functions#1148]
Feature parity with Windows for Metrics emitted by the runtime in Linux Consumption

**Comment by balag0** [comment]
Related - https://github.com/Azure/azure-functions-python-worker/issues/359

**Comment by balag0** [comment]
Bugs:
1) Live metrics doesn't show CPU, memory metrics.
2) Functions monitor tab doesn't show all instances of the app

**Comment by paulbatum** [comment]
Regarding live metrics, I think this is blocked on App Insights support for performance counter style info on Linux. See the discussion in this issue:
https://github.com/Azure/azure-functions-host/issues/3852

**Comment by ericdrobinson** [comment]
>Regarding live metrics, I think this is blocked on App Insights support for performance counter style info on Linux. See the discussion in this issue: Azure/azure-functions-host#3852

@lmolkova mentioned in that discussion that Linux support was in the backlog but not prioritized. Is there a specific GitHub issue tracking that or is it in some other private system?

**Comment by chintanr97** [comment]
Hi, I recently configured tests on Azure Durable Functions based on Consumption Plan, running Linux hosts. I can see that the metrics like memory working set seem to be not working or at least values appear to be zero. The similar function execution on Premium plan is reporting the memory working set correctly. Also, apart from the Live Stream of the execution flow, can I compute CPU usage and Memory Consumed: (1) Per Execution of Activity Function; (2) Per Instance used during the test?

Regards,
Chintan Rajvir

**Comment by balag0** [comment]
Yes, Live stream of CPU and memory usage per instance is on the roadmap.
